src/main/kotlin/com/github/avrokotlin/avro4k/schema/ClassSchemaFor.kt

```diff
@@ -115,18 +116,20 @@ class ClassSchemaFor(
          else -> schemaOrFixed.overrideNamespace(ns)
       }
 
-      val default: Any? = annos.default()?.let {
+      val default: Any? = annos.default()?.let { annotationDefaultValue ->
```

Comment:
Maybe we should extract the logic for determining the default value to a separate function. With the added if clause, it becomes hard to read.

Reply:
Good point, I will do it!

Reply:
Done